docs/publish/publish-add-in-vs-code.md

```diff
@@ -12,9 +12,44 @@ This article describes how to publish an Office Add-in that you created using th
 > [!NOTE]
 > For information about publishing an Office Add-in that you created using Visual Studio, see [Publish your add-in using Visual Studio](package-your-add-in-using-visual-studio.md).
 
+## Publishing pre-requisites
+
+An Office Add-in consists of a web application and a manifest file. The web application defines the add-in's user interface and functionality, while the manifest specifies the location of the web application and defines settings and capabilities of the add-in.
+
+While you're developing your add-in, you can run the add-in on your local web server (`localhost`), but when you're ready to publish it for other users to access, you'll need to deploy the web application and update the manifest to specify the URL of the deployed application.
+
+When your add-in is working as desired and you're ready to publish it for other users to access, ensure you have the following pre-requisites:
+
+- An [Azure account](https://azure.microsoft.com/free/). You can start one for free if you don't already have an account.
+- An Azure storage account. If you need to set one up after creating your Azure account, do so using instructions in [Create an Azure Storage Account](https://docs.microsoft.com/azure/developer/javascript/tutorial-vscode-static-website-node-03).
+
 ## Publishing an add-in for other users to access
 
-[!include[instructions for publishing an Office Add-in](../includes/publish-add-in.md)]
+1. Open your browser and navigate to https://portal.azure.com/, then select **Storage accounts**. You will see your storage accounts. Select the one you would like to use for your add-in.
+
+2. On this same page under the settings menu, choose the **Static website** option. You'll notice **Index document name** and **Error document path** are pre-populated with the filename index.html. Change both fields from index.html to **taskpane.html**. Next, copy the **primary endpoint** URL that you did not change.
+
+![Static website settings in Azure](static-website-in-azure.png)
```

Comment:
I think the image paths in this article are incorrect. You may need to do something like:
```suggestion
![Static website settings in Azure](../images/static-website-in-azure.png)
```

Reply:
I'm gonna commit to see if these work, will fix if that doesn't fix it